app/services/search/page_searcher.rb

```diff
@@ -0,0 +1,55 @@
+class Search::PageSearcher
+
+  def self._search(string)
+    CampaignPage.joins(:widgets).
+        where("campaign_pages.title ILIKE ? OR widgets.content #>> '{text_body_html}' ILIKE ?", "%#{string}%", "%#{string}%")
+  end
```

Comment:
I'm not using it right now, but I thought it'd be good to keep it in at least as a template for a method that can be used to get campaign pages by joining by a field in campaign_pages with a jsonb key in widget content. Is that a good enough reason to keep it in?